lib/epics/at-copy-match-to-clipboard.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+
+import {Observable} from 'rxjs'
```

Comment:
not needed, and removed.

( I ignorantly thought that it would be needed because of this: action$: rxjs$Observable<Action> )